<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Academic</class>
 <widget class="QDialog" name="Academic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Academic</string>
  </property>
  <widget class="QWidget" name="Fback" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgb(36, 36, 36);
}</string>
   </property>
   <widget class="QWidget" name="Fback1" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>1221</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgb(45, 45, 45);
}</string>
    </property>
    <widget class="QWidget" name="Fback2" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>1171</width>
       <height>611</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: rgb(55, 55, 55);
	border-radius:20px;
}</string>
     </property>
     <widget class="QWidget" name="btnBack" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>401</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	
	background-color: rgb(65, 65, 65);
	border-radius:10px;
}</string>
      </property>
      <widget class="QComboBox" name="type">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Cascadia Code</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
	background-color:rgb(255, 131, 133);
	border-radius:20px;
	padding-left:10px;
	padding-right:10px;
	color: black;
}</string>
       </property>
       <item>
        <property name="text">
         <string>Lecture</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Assignment</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="btnSchedule">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>330</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgb(255, 213, 215);
	border-radius:25px;
	border-color: rgb(53, 69, 93);
	
	color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Schedule</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_date">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>220</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
       </property>
       <property name="text">
        <string>Year/Month/Date</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_type">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnClear">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>240</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgb(255, 213, 215);
	border-radius:25px;
	border-color: rgb(53, 69, 93);
	
	color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSelected">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgb(255, 213, 215);
	border-radius:25px;
	border-color: rgb(53, 69, 93);
	
	color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Clear Selected</string>
       </property>
      </widget>
      <widget class="QLabel" name="txtAcademic">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>341</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poor Richard</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
       </property>
       <property name="text">
        <string>Academic Schedule</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnBack_2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>420</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgb(152, 96, 100);
	border-radius:25px;
	border-color: rgb(53, 69, 93);
	
	color: gold;
}</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>240</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Cascadia Code</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit{
	background-color:rgb(255, 131, 133);
	border-radius:20px;
	padding-left:10px;
	padding-right:10px;
	color:black;
}</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
       </property>
       <property name="displayFormat">
        <string>yyyy/MM/dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2024</year>
         <month>11</month>
         <day>10</day>
        </date>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>330</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Cascadia Code</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QTimeEdit{
	background-color:rgb(255, 131, 133);
	border-radius:20px;
	padding-left:10px;
	padding-right:10px;
	color:black;
}</string>
       </property>
       <property name="displayFormat">
        <string>hh:mm AP</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_time">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>310</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
       </property>
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
      <widget class="QComboBox" name="subjectCombo">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>420</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Cascadia Code</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
	background-color:rgb(255, 131, 133);
	border-radius:20px;
	padding-left:10px;
	padding-right:10px;
	color: black;
}</string>
       </property>
       <item>
        <property name="text">
         <string>ICT1306</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ICT1305</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ICT1402</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ICT1308</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_subject">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>400</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(255, 213, 215);
}</string>
       </property>
       <property name="text">
        <string>Subject</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tableBack" native="true">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>681</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	
	background-color: rgb(65, 65, 65);
	border-radius:10px;
}</string>
      </property>
      <widget class="QTableWidget" name="tableAcademic">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>631</width>
         <height>511</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Cascadia Code</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Subject</string>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
         </font>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
